spec/storage.spec.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+describe("storage", () => {
+    var containerMock,create;
+
+    beforeEach(() => {
+        containerMock = jasmine.createSpyObj("containerMock", [
+            "call"
+        ]);
+        create = (config) => {
+            return require("../lib/storage")(config, containerMock);
+        };
+    });
+    it("finds a storage engine", () => {
+        expect(() => {
+            create({
+                storage: {
+                    engine: "s3"
+                }
+            });
+        }).not.toThrow();
+        expect(containerMock.call).toHaveBeenCalled();
+    });
+    it("has no config options", () => {
```

Comment:
Try to word this sentence to describe the behavior you expect.

The storage .....

Your test says this:  The storage has no config options.  That's not behavior.  Try something like "will error without a proper configuration".

This comment applies to the other tests.